encoding.bs

```diff
@@ -1975,26 +1975,26 @@ constructor steps are:
 
 <ol>
  <li>
-  <p>If <var>encoder</var>'s <a>pending high surrogate</a> is non-null, then:
+  <p>If <var>encoder</var>'s <a>leading surrogate</a> is non-null, then:
 
   <ol>
-   <li><p>Let <var>high surrogate</var> be <var>encoder</var>'s <a>pending high surrogate</a>.
+   <li><p>Let <var>leadingSurrogate</var> be <var>encoder</var>'s <a>leading surrogate</a>.
 
-   <li><p>Set <var>encoder</var>'s <a>pending high surrogate</a> to null.
+   <li><p>Set <var>encoder</var>'s <a>leading surrogate</a> to null.
 
-   <li><p>If <var>item</var> is in the range U+DC00 to U+DFFF, inclusive, then return a scalar value
-   whose value is 0x10000 + ((<var>high surrogate</var> &minus; 0xD800) &lt;&lt; 10) +
-   (<var>item</var> &minus; 0xDC00).
+   <li><p>If <var>item</var> is a <a>trailing surrogate</a>, then return a scalar value whose value
+   is
+   0x10000 + ((<var>leadingSurrogate</var> &minus; 0xD800) &lt;&lt; 10) + (<var>item</var> &minus; 0xDC00).
 
    <li><p><a>Prepend</a> <var>item</var> to <var>input</var>.
 
    <li><p>Return U+FFFD.
   </ol>
 
- <li><p>If <var>item</var> is in the range U+D800 to U+DBFF, inclusive, then set <a>pending high
- surrogate</a> to <var>item</var> and return <a>continue</a>.
+ <li><p>If <var>item</var> is a <a>leading surrogate</a>, then set <a>leading surrogate</a> to
```

Comment:
I think both these `<a>leading surrogate</a>` will link to the same place, which seems wrong.

Reply:
Should be fixed once https://github.com/whatwg/infra/commit/0dba3a29da6e071a30e06bb837e282eca33efbc9 is indexed, but I should probably use explicit `for` as well.